config/install.yaml

```diff
@@ -900,7 +900,7 @@ spec:
     - jsonPath: .status.phase
       name: Phase
       type: string
-    - jsonPath: .spec.replicas
+    - jsonPath: .status.desiredReplicas
       name: Desired
```

Comment:
not related to my change but codegen was failing - I guess numaflow changed its CRD